src/extension.ts

```diff
@@ -10,14 +10,14 @@ import * as fs from 'fs';
 import * as path from 'path';
 import { MongoExplorer } from './mongo/explorer';
 import { MongoCommands } from './mongo/commands';
-import { Model, Database, Server, IMongoResource, MongoScript, Collection } from './mongo/mongo';
+import { Model, Database, Server, IMongoResource, MongoCommand, Collection } from './mongo/mongo';
```

Comment:
I'm not familiar with the codebase so these might be dumb questions: can MongoScript do what MongoCommand does (or is MongoScript a superset of MongoCommand in terms of functionality)?

Is it possible that in the future "Execute Script" will be replacing "Executing Command", so someone will have to change this all back?

If yes, is it possible to simply change the UI text to just say "Command" instead of "Script"?